### PR TITLE
Add license header to m2.conf

### PR DESCRIPTION
**Pull request by XenoAmess**
...just found no reason not adding it, as all other scripts/shell have added, and this file format actually can accept \# comments...

### CODE REVIEW

**Comment by cstamas** [comment]
Perso I'd rather fix spotless on BOM, as we still want BOM formatted (instead as in this PR, skipping it) @gnodet ?

File: maven-bom/pom.xml [hunk 1]
```diff
@@ -204,6 +204,14 @@ under the License.
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>com.diffplug.spotless</groupId>
+        <artifactId>spotless-maven-plugin</artifactId>
```
Comment (cstamas) [inline maven-bom/pom.xml:209]: This is a no go

**Review by cstamas — CHANGES_REQUESTED** [review]

File: maven-bom/pom.xml (hunk 1 above)
Comment (XenoAmess) [reply maven-bom/pom.xml]: @cstamas removed. hope you find another better way for resolving this.

**Comment by gnodet** [comment]
> Perso I'd rather fix spotless on BOM, as we still want BOM formatted (instead as in this PR, skipping it) @gnodet ?

Which BOM is broken ?

**Review by gnodet — APPROVED** [review]

**Pull request merged**